kllawesome.py

```diff
@@ -0,0 +1,66 @@
+# imports from third party packages
```

Comment:
If this is the main project maybe throw in a (assuming python 3)
`#!/usr/bin/env python3` At the top of the file

Reply:
Since we don't run it as a shell script we don't need the shebang.